musl doesn't have isnanf, and this fixes an implicit-function-decl
how did this even compile without failing at link time?
rest for implicit
--
--- openjdk/jdk/src/solaris/native/common/jdk_util_md.h
+++ openjdk/jdk/src/solaris/native/common/jdk_util_md.h
@@ -37,7 +37,7 @@
 #define ISNAND(d) isnan(d)
 #elif defined(__linux__) || defined(_ALLBSD_SOURCE)
 #include <math.h>
-#define ISNANF(f) isnanf(f)
+#define ISNANF(f) isnan(f)
 #define ISNAND(d) isnan(d)
 #elif defined(_AIX)
 #include <math.h>
